### ISSUE TITLE
Enable model to directly work with .tiff files

### ISSUE THREAD

**Issue by rroelz** [janablechschmidt/MetaRange.jl#28]
Is there a way to directly work with TIFF files? Kinda comparable to Rs `terra`. This isn't highest priority. 
This is a consideration for input and output.

**Comment by janablechschmidt** [comment]
[SimpleSDMDatasets](https://github.com/PoisotLab/SpeciesDistributionToolkit.jl/tree/main/SimpleSDMDatasets#simplesdmdatasets) might be worth checking out to work with raster files directly

**Comment by janablechschmidt** [comment]
ooh it's even possible to get raster files directly from CHELSA with [RasterDataSources.jl](https://github.com/EcoJulia/RasterDataSources.jl), how cool!

**Comment by rroelz** [comment]
Yeah there's a lot of nice things. The trouble is it would need a MAJOR rework. Probably not impossible but would need to throw quite a few things around
